Source/Renderer/Buffer.js

```diff
@@ -6,7 +6,8 @@ define([
         '../Core/destroyObject',
         '../Core/DeveloperError',
         '../Core/IndexDatatype',
-        './BufferUsage'
```

Comment:
In order to get more 3D Tiles code into master sooner, can you open a separate pull request into master for these changes and the changes to ShaderProgram.js?